examples/resnet-runtime.cpp

```diff
@@ -38,15 +38,22 @@ llvm::cl::opt<std::string>
                                   "png's with standard imagenet normalization"),
                    llvm::cl::init("../tests/images/imagenet/"),
                    llvm::cl::Positional, llvm::cl::cat(category));
-llvm::cl::opt<unsigned> numDevices("numDevices",
+llvm::cl::opt<unsigned> numDevices("num-devices",
                                    llvm::cl::desc("Number of Devices to use"),
                                    llvm::cl::init(5), llvm::cl::value_desc("N"),
                                    llvm::cl::cat(category));
 llvm::cl::opt<unsigned>
-    maxImages("maxImages",
+    maxImages("max-images",
               llvm::cl::desc("Maximum number of images to load and classify"),
-              llvm::cl::init(1000), llvm::cl::value_desc("N"),
+              llvm::cl::init(100), llvm::cl::value_desc("N"),
               llvm::cl::cat(category));
+llvm::cl::opt<std::string>
+    writeTrace("dump-trace", llvm::cl::desc("Write trace logs to disk"),
```

Comment:
This sounds more like a name you'd give to a boolean option. Consider `tracefile-path` or something like that.